### PR TITLE
Bugfix - Touch delay in iOS

### PR DESCRIPTION
**Pull request by jutaz**
Due to use of UITapGestureRecognizer and it depending on a double-tap gesture, it caused a ~400ms delay in the tap actually being registered.

This fixes that by shortening the default delay to 200ms.

### CODE REVIEW

File: ios/RCTMGL/RCTMGLMapViewManager.m [hunk 1]
```diff
@@ -41,27 +65,28 @@ - (UIView *)view
...
+    UIShortTapGestureRecognizer *doubleTap = [[UIShortTapGestureRecognizer alloc] initWithTarget:self action:nil];
     doubleTap.numberOfTapsRequired = 2;
-    
-    UITapGestureRecognizer *tap = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(didTapMap:)];
+
+    UIShortTapGestureRecognizer *tap = [[UIShortTapGestureRecognizer alloc] initWithTarget:self action:@selector(didTapMap:)];
+
     [tap requireGestureRecognizerToFail:doubleTap];
-    
+
```
Comment (blainsadler) [inline ios/RCTMGL/RCTMGLMapViewManager.m:74]: should we set up an editor config that uses their style?

File: ios/RCTMGL/RCTMGLMapViewManager.m [hunk 2]
```diff
@@ -497,21 +522,20 @@ - (BOOL)mapView:(MGLMapView *)mapView annotationCanShowCallout:(id<MGLAnnotation
...
     ((RCTMGLMapView *) mapView).isUserInteraction = (BOOL)(reason & ~MGLCameraChangeReasonProgrammatic);
-    
+
     NSDictionary *payload = [self _makeRegionPayload:mapView animated:animated];
     [self reactMapDidChange:mapView eventType:RCT_MAPBOX_REGION_WILL_CHANGE_EVENT andPayload:payload];
 }
 
 - (void)mapViewRegionIsChanging:(MGLMapView *)mapView
 {
-    NSDictionary *payload = [self _makeRegionPayload:mapView animated:false];
```
Comment (blainsadler) [inline ios/RCTMGL/RCTMGLMapViewManager.m]: why did these change?

**Comment by blainsadler** [comment]
Nice work, can't wait to try this out!

File: ios/RCTMGL/RCTMGLMapViewManager.m (hunk 1 above)
Comment (jutaz) [reply ios/RCTMGL/RCTMGLMapViewManager.m:74]: Trailing whitespaces aren't really _ever_ a style TBH. I can remove them, but it'll always be a diff-picking job to push up changes.

I think the future for this repo will be that we'll start pushing to it more, so I think we should consider setting it up for our own style. WDYT?

File: ios/RCTMGL/RCTMGLMapViewManager.m (hunk 2 above)
Comment (jutaz) [reply ios/RCTMGL/RCTMGLMapViewManager.m]: Whoops, missed that - will remove.

**Review by mike-guerrette — APPROVED** [review]
LGTM 👍 

Will test this out right away

**Pull request merged**